### PR TITLE
fix issue #5299

### PR DESCRIPTION
**Pull request by oneseedfruit**

### CODE REVIEW

File: scene/animation/animation_player.cpp [hunk 1]
```diff
@@ -572,7 +572,7 @@ void AnimationPlayer::_animation_process_data(PlaybackData &cd,float p_delta,flo
 
 	} else {
 		
-		if (next_pos<0 or next_pos>len) {
+        if (next_pos<0 || next_pos>len) {
```
Comment (akien-mga) [inline scene/animation/animation_player.cpp]: The fix is good, but you introduced indentation errors :)

Also, no need to mention the files and lines in the commit log, it's a bit redundant with the diff that shows them already.

**Comment by volzhs** [comment]
you should need to change spaces to tab

File: scene/animation/animation_player.cpp (hunk 1 above)
Comment (ranmaru90) [inline scene/animation/animation_player.cpp]: Oops, sorry! I'm not sure how did the indentation errors happened when I only changed or to || and and to &&. :(

Comment (akien-mga) [inline scene/animation/animation_player.cpp]: Most likely your IDE is configured for spaces instead of tabs, and enforces it on all lines changed.

**Comment by vnen** [comment]
Thanks for the fix. I pushed 5cb31f6d5b667a551092b699de6f4786e0bb5b6d before seeing this, so it's already done.

File: scene/animation/animation_player.cpp (hunk 1 above)
Comment (ranmaru90) [inline scene/animation/animation_player.cpp]: Thanks for the hints.

I'm using QtCreator, so it's Tools -> Options -> Text Editor -> Behavior and under "Tab Policy", change to "Tabs Only". Am I right?

**Pull request closed without merging**